auto_nag/scripts/regression.py

```diff
@@ -46,16 +47,14 @@ def get_bz_params(self, date):
             'f5': 'longdesc',
             'o5': 'changedbefore',
             'v5': end_date,
-            # Ignore bugs filed by the intermittent failure bot.
-            'emailtype1': 'notequals',
-            'email1': 'intermittent-bug-filer@mozilla.bugs',
-            'emailreporter1': 1,
-            # And by the wpt-sync bot.
-            'emailtype2': 'notequals',
-            'email2': 'wptsync@mozilla.bugs',
-            'emailreporter2': 1,
         }
 
+        for i, email in enumerate(utils.get_config(self.name(), 'reporter_blacklist')):
```

Comment:
'f6': 'reporter', 'o6': 'nowords', 'v6': ','.join(blacklist) should be better

Reply:
Done